src/main/java/com/forge/revature/controllers/PortfolioController.java

```diff
@@ -101,11 +106,162 @@ public Portfolio getByID(@PathVariable(name = "id") int id){
     public Portfolio postPort(@RequestBody Portfolio port){
         return portRepo.save(port);
     }
+    
```

Comment:
Why is this all being done in a controller class and not a service class. Isn't this business logic?